src/renderer/components/+config-priority-classes/priority-classes.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+import "./priority-classes.scss";
+
+import * as React from "react";
+import { observer } from "mobx-react";
+import { priorityClassStore } from "./legacy-store";
+import type { PriorityClass } from "../../../common/k8s-api/endpoints/priority-class.api";
+import { KubeObjectStatusIcon } from "../kube-object-status-icon";
+import type { KubeObjectDetailsProps } from "../kube-object-details";
+import { KubeObjectListLayout } from "../kube-object-list-layout";
+import { SiblingsInTabLayout } from "../layout/siblings-in-tab-layout";
+import { KubeObjectAge } from "../kube-object/age";
+
+enum columnId {
+  name = "name",
+  value = "value",
+  globalDefault = "global-default",
+  age = "age",
+}
+
+export interface PriorityClassesProps extends KubeObjectDetailsProps<PriorityClass> {
+}
+
+@observer
+export class PriorityClasses extends React.Component<PriorityClassesProps> {
+  render() {
+    return (
+      <SiblingsInTabLayout>
+        <KubeObjectListLayout
+          isConfigurable
+          tableId="configuration_priority_classes"
+          className="PriorityClasses"
+          store={priorityClassStore}
```

Comment:
This entire component could probably be done using `withInjectables` so that this store doesn't have to use the legacy globals.

Reply:
Again: all other components don't use `withInjectables` then this change would be out of the scope for this small feature.

Reply:
Ok I refactored

Reply:
Thanks!